hwcomposer/qeglfsscreen.h

```diff
@@ -85,6 +85,9 @@ class QEglFSScreen : public QPlatformScreen //huh: FullScreenScreen ;) just to f
     Qt::ScreenOrientation orientation() const;
 #endif
 
+    QPlatformScreen::PowerState powerState() const;
+    void setPowerState(QPlatformScreen::PowerState state);
```

Comment:
Missing from the earlier ones too, but could have 'override' set for these. Nicely ensures that it's actually reimplementing an API.